06_puppy/lexual/puppy.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sync"
+)
+
+var out io.Writer = os.Stdout
+
+type Puppy struct {
+	ID     uint32
+	Breed  string
+	Colour string
+	Value  int
+}
+
+// Storer defines standard CRUD operations for Puppys
```

Comment:
nitpick: doc strings are English sentences, so they should end with a full stop. See https://blog.golang.org/godoc-documenting-go-code for details. Please fix this throughout.